geom/encoding/geojson/geojson_test.go

```diff
@@ -0,0 +1,115 @@
+package geojson_test
+
+import (
+	"encoding/json"
+	"reflect"
+	"testing"
+
+	"github.com/terranodo/tegola/geom"
+	"github.com/terranodo/tegola/geom/encoding/geojson"
+)
+
+func TestFeatureMarshalJSON(t *testing.T) {
+	type tcase struct {
+		geom     geom.Geometry
+		expected []byte
+	}
+
+	fn := func(t *testing.T, tc tcase) {
+		t.Parallel()
+
+		f := geojson.Feature{
+			Geometry: geojson.Geometry{tc.geom},
+		}
+
+		output, err := json.Marshal(f)
+		if err != nil {
+			t.Error(err)
+			return
```

Comment:
Let the test continue to see in one shot if other test cases fail?

Reply:
It seems all these tests are expecting good cases. In that case, I think it makes more sense to bail if there is an error Marshalling. 

It would be better to add an error element to the `tcase struct`, so that we can have bad cases.

Reply:
I will add an invalid geom test too.